etc/config/odin.amazon.properties

```diff
@@ -0,0 +1,13 @@
+compilers=&odin
+defaultCompiler=odin2209
+group.odin.compilers=odin2209:odin2208
+group.odin.objdumper=/opt/compiler-explorer/gcc-11.1.0/bin/objdump
+group.odin.demangler=
+group.odin.isSemVer=false
+group.odin.baseName=odin
```

Comment:
No need for `baseName` if the compiler is not semver'ed. But `groupName` is probably one you want to add for it to show up in the header of the compiler selector